### PR TITLE
Fix the typo in deployment-and-usage.md

### LINKED ISSUE #567: Typing error in https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/docs/get-started/deployment-and-usage.md
In https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/docs/get-started/deployment-and-usage.md,
under the section **NFD-Worker**, 
`This assures re-labeling on regular intervals capturing changes in the system configuration and mames sure that new nodes are labeled as they are added to the cluster.`

`mames` shoud be written as make

### PR DESCRIPTION
**Pull request by vaibhav2107**
Fixed typo in https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/docs/get-started/deployment-and-usage.md

Fixes #567

### CODE REVIEW

**Comment by vaibhav2107** [comment]
Hi @adrianchiris @zvonkok 
Please have a look.

File: docs/get-started/deployment-and-usage.md [hunk 1]
```diff
@@ -285,7 +285,7 @@ labels. The provided template will configure these for you.
 
 NFD-Worker is preferably run as a Kubernetes DaemonSet. This assures
 re-labeling on regular intervals capturing changes in the system configuration
-and mames sure that new nodes are labeled as they are added to the cluster.
+and make sure that new nodes are labeled as they are added to the cluster.
```
Comment (marquiz) [inline docs/get-started/deployment-and-usage.md:288]: I think this should be *makes* instead of *make*, right(?)

Comment (adrianchiris) [reply docs/get-started/deployment-and-usage.md:288]: +1

Comment (vaibhav2107) [reply docs/get-started/deployment-and-usage.md:288]: Thanks @marquiz , @adrianchiris for suggesting the change, will update this

**Comment by vaibhav2107** [comment]
@marquiz @adrianchiris 
PTAL!

**Review by marquiz — APPROVED** [review]
/label tide/merge-method-squash
/lgtm

**Pull request merged**